overlord/ifacestate/ifacemgr.go

```diff
@@ -217,6 +223,8 @@ func (m *InterfaceManager) initUDevMonitor() error {
 		mon.Disconnect()
 		return err
 	}
+	m.enumeratedDeviceKeys = make(map[string]map[string]bool)
```

Comment:
what's the lock used to access these?  afaict we are holding the state lock in places where we use this but not here

Reply:
Good catch, moved initialization of those to the Manager(). Thanks